test/ref_ops_test.cpp

```diff
@@ -1330,6 +1339,56 @@ TEST_CASE(div_test)
     EXPECT(migraphx::verify_range(results_vector, gold));
 }
 
+TEST_CASE(div_zero_compile_trap_after_passes)
+{
+    migraphx::program p;
+    auto* mm  = p.get_main_module();
+    auto zero = mm->add_literal(0.0f);
+    auto x    = mm->add_parameter("x", {migraphx::shape::float_type, {1}});
+    mm->add_instruction(migraphx::make_op("div"), x, zero);
+    run_pass(*mm);
+
+    bool result = false;
+    try
+    {
+        p.compile(migraphx::ref::target{});
```

Comment:
You can use `EXPECT(test_throws())`

Reply:
Will add this

Reply:
`EXPECT(test::throws([&]{p.compile(migraphx::ref::target{});}));`

Reply:
Done